Chukwunonso/new.js

```diff
@@ -0,0 +1 @@
+console.log('my name is Chukwunonso');
```

Comment:
Kindly add a newline at the end of the file.

Reply:
ok, I will do that right away. thanks.